spring-cloud-kubernetes-fabric8-config/src/test/java/org/springframework/cloud/kubernetes/fabric8/config/LabeledSecretContextToSourceDataProviderTests.java

```diff
@@ -446,4 +451,57 @@ void testYaml() {
 		Assertions.assertEquals(sourceData.sourceName(), "secret.color-secret.default");
 	}
 
+	/**
```

Comment:
a test that proves that log statements are on par with caching, first request is from k8s, subsequent ones are from cache